docs/Splashkit/Applications/npm-installation-guide.md

```diff
@@ -0,0 +1,153 @@
+# Npm installation guide: Windows and Mac
+
+###### (scroll down for Mac instructions)
```

Comment:
**suggestion:** We might want to avoid using header semantics for subtitles, especially not to jump to order (for example, h1, then h2, not h5). I personally don't think we need this comment but don't have a strong opinion against it.

```suggestion
_(scroll down for Mac instructions)_
```